HWLesson6_8/main.js

```diff
@@ -0,0 +1,137 @@
+// const cleanerListEl = document.querySelector(".js-cleaners-list");
+// function displayCleaner(cleaner) {
+//     const cleanerEl = document.createElement("p");
+//     cleanerEl.innerHTML= `<h2>${cleaner.name}</h2>
+//         <p>Rating: ${cleaner.rating}</p>
+//         <p>Capacität: ${cleaner.optional.liter} L</p>
+//         <p>Faber: ${cleaner.optional.color}</p>
+//         <p>Quelle: ${cleaner.optional.electricSource}</p>
+//         <p>Technologie: ${cleaner.optional.technology}</p>
+//         `;
+//     cleanerListEl.appendChild(cleanerEl);
+// }
+
+        //  Display products image & description 
+const cleanerListEl = document.getElementById("bosch");
+const cleanerListEl1 = document.getElementById("rowenta");
+const cleanerListEl2 = document.getElementById("dyson");
+const cleanerListEl3 = document.getElementById("aeg");
+
+cleanerListEl.innerHTML = `<h3>${BoschCln.name}</h3>

```

Comment:
To avoid repetitive code blocks, you can do this using a function and for loops to display each item.